src/Socket.php

```diff
@@ -680,15 +683,20 @@ public static function showListForItem($item)
        // Link to open a new socket
         if ($item->getID() && self::canCreate()) {
             echo "<div class='firstbloc'>";
-            Html::showSimpleForm(
-                Socket::getFormURL(),
-                '_add_fromitem',
-                __('New socket for this item...'),
-                [
-                    '_from_itemtype' => $item->getType(),
-                    '_from_items_id' => $item->getID(),
-                ]
-            );
+            echo "\n<form method='get' action='" . Socket::getFormURL() . "'>\n";
+            echo "<input type='hidden' name='items_id' value='" . $item->getID() . "'>\n";
+            echo "<input type='hidden' name='itemtype' value='" . $item->getType() . "'>\n";
+            echo "<div class='firstbloc'><table class='tab_cadre_fixe'>\n";
+            echo "<tr class='tab_bg_2'>";
+            echo "<td class='tab_bg_2 center'>";
+            echo "<td>\n";
+            echo "<input type='submit' name='add_several' value=\"" . __('New socket for this item...') . "\" class='btn btn-primary'>\n";
+            echo "</td>";
+            echo "<td>\n";
+            echo __('Add several sockets');
+            echo "&nbsp;<input type='checkbox' name='several' value='1'></td>\n";
+            echo "</tr></table></div>\n";
+            Html::closeForm();
```

Comment:
With this method, CSRF token is added in URL parameters. You should probably use `</form>` form directly.

Reply:
done